task_templates/estimators/python_anomaly/model-metadata.yaml

```diff
@@ -1,26 +1,36 @@
+# model-metadata.yaml allows to specify task input/output requirements to help validate a blueprint where it's used.
+# It's also valuable if you are using `drum push`
 # see MODEL-METADATA.md and VALIDATION-SCHEMA.md for full explanations of fields
 name: python_anomaly
-type: training
-targetType: anomaly
-# modelID: optional, used with drum push to create a new version of an existing model
-# environmentID: optional, used with drum push to select environment
-#trainingModel:
-#  trainOnProject: optional project on which to train
-validation:
-  # Path is relative to this file
-  input: ../../../tests/testdata/iris_binary_training.csv
-  targetName: MEDV
+type: training # training (for custom tasks) or inference (for custom inference models)
+environmentID: 5e8c889607389fe0f466c72d # optional, only required when using "drum push" to select environment
+targetType: anomaly # can be one of: transform, binary, regression, anomaly, multiclass, unstructured (unstructured is only available for inference models)
+
+# This part below defines the task's schema. Using this schema, on the fly,
+# DataRobot will validate if the task requirements match neighboring tasks in a blueprint
+# Note that it's only available for tasks, and not for inference models
+# Every condition (and the whole section) is optional
 typeSchema:
+  # Specify what data types this task allows as input
```

Comment:
@drenfr01 Seems like this line got here accidently. There is not such a line in the example yaml. Need to be fixed in all files.